intellij-ext/modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordApi.kt

```diff
@@ -42,7 +44,7 @@ object MirrordApi {
     private const val cliBinary = "mirrord"
     const val targetlessTargetName = "No Target (\"targetless\")"
     private val logger = MirrordLogger.logger
-    private fun cliPath(wslDistribution: WSLDistribution?): String {
```

Comment:
I know there was no line space earlier but might worth inserting since it's a bit crowded